docs/www/configuration.md

```diff
@@ -625,6 +625,19 @@ rpk:
   # Default: ''
   coredump_dir: "/var/lib/redpanda/coredump"
 
+  # Creates a "ballast" file that can be removed in the odd case that a redpanda node
+  # runs out of space, so that the node can be restarted and troubleshot.
```

Comment:
The node will not need to be restarted, instead this will free up space so that other redpanda commands can be run (delete topic, etc.).